### PR TITLE
docs: PDE updates

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR adds documentation for a handful of PDE features:

- Document how to download application log files.
- Document the deprecation of the ability to enable virtual clusters on host clusters.
- Moved cluster groups to the cluster heading.
- Warned cluster group admins about setting defaults lower than the required minimum values for a virtual cluster.

💻 [Preview](https://deploy-preview-1120--docs-spectrocloud.netlify.app/)

- [Logs Page](https://deploy-preview-1120--docs-spectrocloud.netlify.app/devx/apps/logs)
- [Resource Quotas](https://deploy-preview-1120--docs-spectrocloud.netlify.app/devx/resource-quota)
- [Cluster Groups Relocated](https://deploy-preview-1120--docs-spectrocloud.netlify.app/clusters/cluster-groups)
- [Create Cluster Group](https://deploy-preview-1120--docs-spectrocloud.netlify.app/clusters/cluster-groups/create-cluster-group)

🎫 [PPD-990](https://spectrocloud.atlassian.net/browse/PPD-990)
🎫 [PPD-1006](https://spectrocloud.atlassian.net/browse/PPD-1006)
🎫 [PPD-1065](https://spectrocloud.atlassian.net/browse/PPD-1065)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1120#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@karl-cardenas-coding: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | f9294d97c8ee0467d946e90051a947e1a075f4c0 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1120/librarium_build/1623842244962291712) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/04.5-devx/06-apps/05-logs.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+* Access to view the application and its logs.
+
+
+## Download Logs
+
+1. Log in to [Palette](https://console.spectrocloud.com)
+
+
+2. Navigate to the **User Menu** and select **Switch to App Mode**. If you are already in app mode view, move to the next step.
```
Comment (ritawatson) [inline content/docs/04.5-devx/06-apps/05-logs.md:50]:
```suggestion
2. If you are not already in App Mode, navigate to the **User Menu** and select **Switch to App Mode**.
```

File: content/docs/04.5-devx/06-apps/05-logs.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+
+3. On the left **Main Menu**, select **Apps**.
+
+
+4. Select you application.
+
+
+5. Click on the **Actions** drop-down Menu.
```
Comment (ritawatson) [inline content/docs/04.5-devx/06-apps/05-logs.md:59]:
```suggestion
5. Click the **Actions** drop-down menu. 
```

File: content/docs/04.5-devx/06-apps/05-logs.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+4. Select you application.
+
+
+5. Click on the **Actions** drop-down Menu. 
+
+
+6. Select **Download Logs**. 
+
+
+7. A message will pop up with the download link.
```
Comment (ritawatson) [inline content/docs/04.5-devx/06-apps/05-logs.md:65]:
```suggestion
7. A message displays with the download link.
```

File: content/docs/04.5-devx/06-apps/05-logs.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+
+8. Click on the download link and review the files.
```
Comment (ritawatson) [inline content/docs/04.5-devx/06-apps/05-logs.md:68]:
```suggestion
8. Click the download link and review the files.
```

File: content/docs/04.5-devx/06-apps/05-logs.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+
+## Validation
+
+To review the downloaded files, inspect your downloads folder on your device. Search for a zip file with the following file format: `[clusterNameHere]-logs-[currentTimeStamp]`.
```
Comment (ritawatson) [inline content/docs/04.5-devx/06-apps/05-logs.md:73]:
```suggestion
To review the logs, locate a zip file with file name format `[clusterNameHere]-logs-[currentTimeStamp]` in the downloads folder on your device.
```

**Review by ritawatson — APPROVED** [review]
LGTM with suggestions. I was able to follow along. 

I might've missed one occurrence of 'meta data' but couldn't find it again. I suggested 'metadata' for one of the occurrences.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1120#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1120#pullrequestreview-1293712734" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by ritawatson — APPROVED** [review]
LGTM with suggestions. I was able to follow along. 

I might've missed one occurrence of 'meta data' but couldn't find it again. I suggested 'metadata' for one of the occurrences.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1120#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1120#pullrequestreview-1294133181" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**